iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/BusAdapter.java

```diff
@@ -32,189 +33,190 @@ protected BusAdapter(final Bus delegate, final MergingViewIndex index) {
     }
 
     @Override
-    public ComponentAdapter getSynchronousComponent() {
-        return getIndex().getComponent(getDelegate().getSynchronousComponent());
+    public Iterable<TwoWindingsTransformer> getTwoWindingsTransformers() {
+        return getTwoWindingsTransformerStream().collect(Collectors.toSet());
```

Comment:
To be discussed: I have a doubt about the Collectors.toSet():
- are we sure that we have to filter duplicates (if there are duplicates)
- what about the order

Reply:
OK, my mistake

Reply:
Replaced by Iterables.transform method usage